src/test/java/racing/domain/RaceResultTest.java

```diff
@@ -10,9 +10,56 @@ public class RaceResultTest {
     @Test
     void 경주_결과에_라운드_결과를_추가할_수_있다() {
         RaceResult raceResult = new RaceResult();
-        RoundResult roundResult = new RoundResult(List.of(1, 2, 3));
+
+        RoundResult roundResult = new RoundResult(List.of(
+                new CarResult(new CarName("자동차1"), new Position(1)),
+                new CarResult(new CarName("자동차2"), new Position(2)),
+                new CarResult(new CarName("자동차3"), new Position(3))
+        ));
 
         raceResult.addResult(roundResult);
         assertThat(raceResult.getRaceResult()).hasSize(1);
     }
+
+    @Test
+    void 마지막_라운드에서_위치가_가장_멀리_간_자동차가_우승자가_된다() {
+        RaceResult raceResult = new RaceResult();
+
+        RoundResult round1 = new RoundResult(List.of(
+                new CarResult(new CarName("자동차1"), new Position(1)),
+                new CarResult(new CarName("자동차2"), new Position(0))
+        ));
+        RoundResult round2 = new RoundResult(List.of(
+                new CarResult(new CarName("자동차1"), new Position(2)),
+                new CarResult(new CarName("자동차2"), new Position(1))
+        ));
+
+        raceResult.addResult(round1);
+        raceResult.addResult(round2);
+
+        List<CarName> winners = raceResult.winners();
+
+        assertThat(winners)
+                .extracting(CarName::value)
+                .containsExactly("자동차1");
+    }
+
+    @Test
+    void 가장_멀리_간_자동차가_여러_대라면_모두_우승자이다() {
+        RaceResult raceResult = new RaceResult();
+
+        RoundResult onlyRound = new RoundResult(List.of(
+                new CarResult(new CarName("자동차1"), new Position(1)),
```

Comment:
```suggestion
                new CarResult("자동차1", 1),
```
위와 같이 CarResult를 생성할 수 있도록 부 생성자를 추가해 구현해 보면 어떨까?
다수의 생성자를 추가해 객체를 생성하는 개발자의 편의성을 높혀본다.